app/assets/javascripts/surveys.js

```diff
@@ -326,9 +326,37 @@
           ]
         },
         allowedOnMobile: true
+      },
+      {
+        identifier: 'Business_iteration 1',
+        surveyType: 'url',
+        frequency: 6,
+        startTime: new Date('July 2, 2018').getTime(),
+        endTime: new Date('July 7, 2018').getTime(),
+        url: 'https://GDSUserResearch.optimalworkshop.com/treejack/wu86k8e2?c={{currentPath}}',
+        templateArgs: {
+          title: 'Help us make things easier to find on GOV.UK',
+          surveyCta: 'Answer 2 quick questions',
+          surveyCtaPostscript: 'This activity will open in a separate window'
+        },
+        activeWhen: {
+          section: ['business/all']
+        },
+        allowedOnMobile: true
+      },
+      {
+      url: 'https://gdsuserresearch.optimalworkshop.com/treejack/dpo18280-0',
```

Comment:
To make it easier to read when skimming through this file, can we follow the pattern of other survey entries above, e.g: `identifier` first, then `surveyType` etc

Reply:
Do we need to set `allowedOnMobile: true`?